preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -45,6 +46,11 @@ def invalid_value_annotation(input_datum, output_field, value_type) -> Processin
     )
 
 
+def valid_authors(authors: str) -> bool:
+    pattern = r'^([a-zA-Z\s\.\-]+,[a-zA-Z\s\.\-]*;)+'
```

Comment:
```suggestion
    pattern = r'^([a-zA-Z\s\.\-]+,[a-zA-Z\s\.\-]*;)+'
```

This prohibits non-ascii characters - we may or may not want to accept them and then replace when submitting onwards to ENA (they definitely only allow ASCII). We can start restrictive though and become permissive later - we should just make this requirement clear in the docs and possibly make a discussion issue to discuss further.

Reply:
We might also want to ensure that initials are terminated by a full stop, otherwise there'll be unnecessary inconsistency (we have that for West Nile right now:

```
Nicole M Feriancek, ..., Chantal B.F. Vogels,
```

Chantal has full stops after initials, the rest do not. We could require single first names to be followed by period (error), or rewrite to put the full stop there (warning), or inversely prohibit full stops altogether which would have the benefit of being more concise. 

The only issue is with single letter names which do occur if we were to accept Chinese characters (we probably shouldn't).

Reply:
I have added an additional format_authors function to add in dots after initials - I will still add some tests now